### PR TITLE
TF-3200 updated the menu options

### PR DESCRIPTION
**Pull request by florentos17**
i am tackling [this issue](https://github.com/linagora/tmail-flutter/issues/3200)

1. i modified `Open in New Tab` to `Open in new tab`
2. the order of the options is now changed to include `Delete folder` as the last option

### CODE REVIEW

**Comment by hoangdat** [comment]
Hi @chrismaster, please 
- run `dart run intl_generator:extract_to_arb --output-dir=./lib/l10n lib/main/localizations/app_localizations.dart` after changing something in app_localization
- please also change it in Fr too

Thank you

File: lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart [hunk 1]
```diff
@@ -60,11 +60,11 @@ mixin MailboxWidgetMixin {
         MailboxActions.markAsRead,
       MailboxActions.move,
       MailboxActions.rename,
-      MailboxActions.delete,
       if (mailbox.isSubscribedMailbox)
         MailboxActions.disableMailbox
       else
-        MailboxActions.enableMailbox
+        MailboxActions.enableMailbox,
+      MailboxActions.delete
```
Comment (chibenwa) [inline lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart:67]:
```suggestion
        MailboxActions.enableMailbox,
        MailboxActions.delete
```

**Review by chibenwa — APPROVED** [review]

File: lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart (hunk 1 above)
Comment (dab246) [reply lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart:67]: The `enableMailbox` action is in an `else` condition so it needs to be indented. The `delete` action does not need to be indented.

**Comment by florentos17** [comment]
> Hi @chrismaster, please
> 
>     * run `dart run intl_generator:extract_to_arb --output-dir=./lib/l10n lib/main/localizations/app_localizations.dart` after changing something in app_localization
> 
>     * please also change it in `Fr, En, It, Ru` too
> 
> 
> Thank you

did you mean chrismaster or me ?
i had already run the command, the commit includes the resulting code
as for the translations, i don't think any is needed since it is already all in lowercase letters for `Fr`, `It` and `Ru` (only the english version was featuring uppercase letters and needed change)

**Comment by dab246** [comment]
> * as for the translations, i don't think any is needed since it is already all in lowercase letters for `Fr`, `It` and `Ru` (only the english version was featuring uppercase letters and needed change)

Also change it in the `intl_en` file

**Comment by florentos17** [comment]
one more approval is required so that we could merge and close this work :)

**Comment by dab246** [comment]
> one more approval is required so that we could merge and close this work :)

@florentos17 Please rebase it

File: lib/l10n/intl_messages.arb [hunk 2]
```diff
@@ -1226,12 +1226,6 @@
     "placeholders_order": [],
     "placeholders": {}
   },
-  "nameToBeDisplayed": "Name to be displayed to recipients",
-  "@nameToBeDisplayed": {
-    "type": "text",
-    "placeholders_order": [],
-    "placeholders": {}
-  },
```
Comment (dab246) [inline lib/l10n/intl_messages.arb:1234]: why do we need delete it?

Comment (florentos17) [reply lib/l10n/intl_messages.arb:1234]: oh you are right, i rebased this file wrong...

**Comment by dab246** [comment]
@florentos17  

- [ ] Please rebase again

**Pull request merged**